admin_manual/configuration_server/bruteforce_configuration.rst

```diff
@@ -121,22 +121,31 @@ Troubleshooting
 Overview
 ~~~~~~~~
 
-On most setups Nextcloud will work out of the box without any issues. If you
-run into a situation where logging in or connecting is often very slow for multiple users, the first
-step is to check your Nextcloud Server logs to see what IP addresses are being detected (you may need 
-adjust your logging to INFO level temporarily to do so). 
+On most setups Nextcloud will work out of the box without any issues. If you run into a situation where 
+logging in or connecting is often very slow for multiple users, the first step is to check your Nextcloud 
+Server logs to see what IP addresses are being detected (you will need  adjust your ``loglevel`` to ``1`` 
+temporarily to do so).
+
+Look for entries that start with any of the following:
+
+- `Bruteforce attempt from` [...]
+- `IP address throttled` [...]
+- `IP address blocked` [...]
 
 If all clients appear to be coming from the same IP address and that IP address happens to be your 
-proxy, you need to review your ``trusted_proxies`` configuration. If it is a common connection point,
-such as a multi-user office location, it is possible whitelisting is appropriate.
+proxy, you need to review your ``trusted_proxies`` configuration. 
+
+If the IP address is a common connection point, such as a multi-user office location, it is possible whitelisting 
+is appropriate.
```

Comment:
```suggestion
If the IP address is a common connection point, such as a multi-user office location, it is possible allow-listing it.
```

Reply:
at last the later half needs adjusting as it's not a correct sentence :S (mostlikely was me)